### ISSUE TITLE
Tab navigation

### ISSUE THREAD

**Issue by AdrianLThomas** [AdrianLThomas/valet.expert#4]
It doesn't work.

**Comment by AdrianLThomas** [comment]
Working fine (in Chrome)- seems to be a Firefox issue

**Comment by AdrianLThomas** [comment]
https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox
